tests/sles4sap/trento/test_hana_restore_stopped.pm

```diff
@@ -37,6 +37,9 @@ sub run {
         filter => $primary_host);
     cluster_wait_status($primary_host, sub { ((shift =~ m/.+DEMOTED.+SOK/) && (shift =~ m/.+PROMOTED.+PRIM/)); });
 
+    my $cypress_test_dir = "/root/test/test";
+    enter_cmd "cd " . $cypress_test_dir;
```

Comment:
Minor issue. Either go with:
```
enter_cmd "cd $cypress_test_dir";
```
or with:
```
enter_cmd 'cd ' . $cypress_test_dir;
```
Don't see the need for a string concatenation operation (`.`) between a constant string and a variable when declaring the constant within double quotes where you can simple include the variable as part of the string and Perl will evaluate and replace it.

Reply:
I have 3 occurrence of the same, I'll fix them in a next PR

Reply:
Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16132